### PR TITLE
Day 10 Assignment Release.

### PR DESCRIPTION
**Pull request by Zaheer-May-2022**
### Please Approve the PR and Merge to the main

Added Day 10 Assignments.

### CODE REVIEW

File: src/classes/DownTrianglePattern.cls [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        {
+            string s='  ';
+            for(integer j=1;j<=9;j++)
+            {
+                if(i>=2 && i<=4){if(j>=1+i && j<=9-i){b=false;}else{b=true;}}else{b=true;}
+                
+                if(j>=i && j<=10-i && b)
+                {
+                    s = s + 'x';
+
```
Comment (deekshithr2694) [inline src/classes/DownTrianglePattern.cls:14]: please dont add extra spaces, please keep idea on it for next time

File: src/classes/DownTrianglePattern.cls [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+                    s = s + 'x';
+                    
+                }
+                else
+                {
+                    s = s + '_';
+                    
+                }
+            }
+
```
Comment (deekshithr2694) [inline src/classes/DownTrianglePattern.cls:22]: there are lot of spaces dont give space after end

File: src/classes/LTrianglePattern.cls [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    * 
+   **
+  ***
+ ****
+*****
+ ****
+  ***
+   **
+    * 
+*/
```
Comment (deekshithr2694) [inline src/classes/LTrianglePattern.cls:37]: good
but also share me a screen shot on mail

Comment (Zaheer-May-2022) [reply src/classes/LTrianglePattern.cls:37]: Code Screen Shot or What ?

File: src/classes/DownTrianglePattern.cls (hunk 1 above)
Comment (Zaheer-May-2022) [reply src/classes/DownTrianglePattern.cls:14]: What type of extra space ?

File: src/triggers/HelloWorldTrigger.trigger [hunk 4]
```diff
@@ -0,0 +1,6 @@
+trigger HelloWorldTrigger on Book__c (before insert) {
+
+   Book__c[] books = Trigger.new;
+
+   MyHelloWorld.applyDiscount(books);
```
Comment (deekshithr2694) [inline src/triggers/HelloWorldTrigger.trigger:5]: you can directly pass the trigger.new no need to assign again list.

**Review by deekshithr2694 — APPROVED** [review]
Are you asked to write the test classes too, if so i am not able to see test classes please check and if need create test class

**Pull request merged**